test/suites/drift/suite_test.go

```diff
@@ -335,5 +337,62 @@ var _ = Describe("Drift", Label("AWS"), func() {
 			}}),
 			Entry("Start-up Taints Drift", "Start-up Taint", test.ProvisionerOptions{StartupTaints: []v1.Taint{{Key: "example.com/another-taint-2", Effect: v1.TaintEffectPreferNoSchedule}}}),
 		)
+		DescribeTable("awsnodetemplate drift", func(fieldName string, awsnodetemplatespec v1alpha1.AWSNodeTemplateSpec) {
+			if fieldName == "InstanceProfile" {
+				By("creating an instance profile")
+				instanceProfileName := awssdk.String("instance-profile-drift")
+				createInstanceProfile := &iam.CreateInstanceProfileInput{
+					InstanceProfileName: instanceProfileName,
+				}
+				By("adding the karpenter role to new instance profile")
+				_, err := env.IAMAPI.CreateInstanceProfile(createInstanceProfile)
```

Comment:
Doesn't this fail with an `AlreadyExists` error if you run this test more than once?

Reply:
It will but that a will not affect the test